tests/appsec/test_blocking_addresses.py

```diff
@@ -3,6 +3,7 @@
 # Copyright 2021 Datadog, Inc.
 
 import json
+import logging
```

Comment:
are we using this import? 
For logging purposes we can use `from utils.tools import logger`

Reply:
I think that I added it for some debugging purpose and forget to delete. good catch. Removing it